.github/workflows/coveralls.yml

```diff
@@ -25,16 +25,14 @@ jobs:
       matrix:
         branches: [
           {
-            branch: OpenSSL_1_1_1-stable
-          }, {
             branch: openssl-3.1,
-            extra_config: enable-fips
+            extra_config: no-afalgeng enable-fips enable-ssl-trace enable-trace enable-zlib enable-rc5 enable-md2 enable-weak-ssl-ciphers enable-ec_nistp_64_gcc_128
```

Comment:
Do we need to do anything additional here to support the zlib dependencies?

Reply:
I believe not. We have enable-zlib with other CI jobs without anything additional installed.